ui/package.json

```diff
@@ -19,9 +19,11 @@
     "rc-checkbox": "^2.1.8",
     "rc-time-picker": "^3.7.2",
     "react": "^16.9.0",
+    "react-checkbox-tree": "^1.6.0",
```

Comment:
This package seems to be less used. Can we use a more popular lib like https://github.com/react-component/tree?

Reply:
As discussed, going with react-checkbox-tree as there are some issues with rc-tree.